lib/services/project-changes-service.ts

```diff
@@ -227,11 +232,8 @@ export class ProjectChangesService implements IProjectChangesService {
 
 			let fileStats = this.$fs.getFsStats(filePath);
 
-			let changed = fileStats.mtime.getTime() >= this._outputProjectMtime || fileStats.ctime.getTime() >= this._outputProjectCTime;
-			if (!changed) {
-				let lFileStats = this.$fs.getLsStats(filePath);
```

Comment:
`getLstStats` and `getFsStats` are different. This code is used when the file/dir is symlink - we need to check the symlink itself (for example if it is recreated) and this is done by `getLsStats`.